src/components/DropdownButton/DropdownButton.tsx

```diff
@@ -106,11 +107,21 @@ const DropdownButton = React.forwardRef<HTMLButtonElement, DropdownButtonProps>(
           <div css={optionsStyle({ menuPosition })(theme)}>
             {items && (
               <List
-                data={items.map((item) => ({ value: item, label: item }))}
-                rowSize={'small'}
-                handleOptionClick={handleOptionClick}
+                label={'dropdown-button'}
+                onSelectionChange={(keys) => {
```

Comment:
maybe make this a named function for performance purposes

Reply:
solved at [`3b16bc4` (#655)](https://github.com/Orfium/orfium-ictinus/pull/655/commits/3b16bc4ae94f7c2e343aaf69ff08fb13226e39d5)